src/test/csi-qingcloud/Chart.yaml

```diff
@@ -13,10 +13,10 @@
 #  limitations under the License.
 apiVersion: v2
 
-appVersion: 1.2.0
+appVersion: 1.2.1
 name: csi-qingcloud
 description: A Helm chart for Qingcloud CSI Driver
-version: 1.2.6
+version: 1.2.7
```

Comment:
do we need upgrade values.yaml for image tags?

Reply:
Not needed, See https://github.com/kubesphere/helm-charts/blob/master/src/test/csi-qingcloud/templates/_helpers.tpl#L6. The image tag equals to chart appversion.